setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='markitdown',
+    version='0.1.0',
+    package_dir={'': 'src'},
+    packages=find_packages(where='src'),
+    install_requires=[
+        'mammoth',
+        'markdownify',
+        'pandas',
+        'pdfminer.six',
+        'python-pptx',
+        'puremagic',
+        'requests',
+        'beautifulsoup4',
+        'pydub',
+        'SpeechRecognition',
+        'youtube_transcript_api',
+    ],
+    entry_points={
+        'console_scripts': [
+            'markitdown=markitdown.__main__:main',
+        ],
+    },
+    author='Adam Fourney',
+    author_email='adamfo@microsoft.com',
+    description='Convert various file formats to markdown',
+    license='MIT',
+    python_requires='>=3.6',
+)
```

Comment:
Since markitdown are using Hatch, the setup.py file is unnecessary. The CLI settings can be found here.

https://github.com/microsoft/markitdown/blob/81e3f24acd0049a59cd2dcb2d01d0a98cc57c734/pyproject.toml#L51-L52

Reply:
Got it. That might be something useful to add to the documentation. 

Do you have any opinions on using argparse to parse the input flags?